app/models/pair_programming_exercise_feedback.rb

```diff
@@ -12,6 +12,32 @@ class PairProgrammingExerciseFeedback < ApplicationRecord
   scope :intermediate, -> { where.not(normalized_score: 1.00) }
   scope :final, -> { where(normalized_score: 1.00) }
 
+  enum comment: {
+    difficulty_easy: 0,
+    difficulty_some_what_easy: 1,
+    difficulty_ok: 2,
+    difficulty_some_what_difficult: 3,
+    difficult_too_difficult: 4,
+  }, _prefix: true
+
+  enum time: {
+    estimated_time_less_5: 0,
+    estimated_time_5_to_10: 1,
+    estimated_time_10_to_20: 2,
+    estimated_time_20_to_30: 3,
+    estimated_time_more_30: 4,
+  }, _prefix: true
+
+  enum reasons: {
```

Comment:
With Rails magic:

```suggestion
  enum reason_work_alone: {
```

And then, we can remove the `reason` prefix from the entries below.